week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,7 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord, thirdWord);
```

Comment:
Great use of string methods!
How could you do it without methods though? :)

Reply:
function concatenate(firstWord, secondWord, thirdWord) {
  // Write the body of this function to concatenate three words together
  return firstWord + secondWord +t hirdWord;
}